src/passes/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ set(passes_SOURCES
   NameList.cpp
   NameTypes.cpp
   NoInline.cpp
+  LLVMNontrappingFPToIntLowering.cpp
```

Comment:
We didn't put LLVM in the name of the bulk memory pass IIRC - should we match the two? I don't feel strongly either way.

Reply:
Done in https://github.com/WebAssembly/binaryen/pull/7082